docs/server/project_setup.rst

```diff
@@ -186,6 +186,12 @@ project form:
 and then provide a different language code in the **Filesystem language code**
 field for those languages whose codes need to be mapped.
 
+
+.. _project_setup#import-translations:
+
+Import the translations
++++++++++++++++++++++++
+
 Creating the project doesn't actually import all the translations to
 Pootle. To do that you need to run :djadmin:`update_stores` on the command line
```

Comment:
If Pootle FS is default, we shouldn't have any `update_stores` anymore for basic instructions.